Use Github Actions / Vercel Deploy Show Error when evaluating SSR module

My Github Actions ~

```
jobs:
  Build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: actions/setup-node@v4
        with:
          node-version: 20
      - run: npm install
      - run: node -v
      - run: npm -v
      - run: npm run build
```

node v20.16.0
npm v10.8.1

get following error when 'Run npm run build'

```
PM [vite] Error when evaluating SSR module /home/runner/work/official-website/official-website/astro.config.mjs: failed to import "astro-icon"
|- file:///home/runner/work/official-website/official-website/node_modules/@iconify/tools/lib/svg/index.mjs:1
import cheerio from 'cheerio';
       ^^^^^^^
SyntaxError: The requested module 'cheerio' does not provide an export named 'default'
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:134:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:217:5)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async nodeImport (file:///home/runner/work/official-website/official-website/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:52[9](https://github.com/winsames-com/official-website/actions/runs/10370461407/job/28708387970#step:7:10)28:15)
    at async ssrImport (file:///home/runner/work/official-website/official-website/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:52786:16)
    at async eval (/home/runner/work/official-website/official-website/astro.config.mjs:[10](https://github.com/winsames-com/official-website/actions/runs/10370461407/job/28708387970#step:7:11):31)
    at async instantiateModule (file:///home/runner/work/official-website/official-website/node_modules/vite/dist/node/chunks/dep-NjL7WTE1.js:52844:5)

[astro] Unable to load your Astro config

Error: Process completed with exit code 1.
```

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "@fontsource-variable/inter": "^5.0.19",
     "astro": "^4.12.2",
     "astro-embed": "^0.7.2",
-    "astro-icon": "^1.1.0",
+    "astro-icon": "^1.1.1",
     "limax": "4.1.0",
     "lodash.merge": "^4.6.2",
     "unpic": "^3.18.0"

